analyses/independent-samples/01-generate-independent-specimens.R

```diff
@@ -54,18 +54,31 @@ if (!dir.exists(out_dir)){
   dir.create(out_dir, recursive = TRUE)
 }
 
-wgs_primary_file <- file.path(out_dir, 
-                              "independent-specimens.wgs.primary.tsv")
-wgs_relapse_file <- file.path(out_dir, 
-                              "independent-specimens.wgs.relapse.tsv")
-wgs_primplus_file <- file.path(out_dir, 
-                              "independent-specimens.wgs.primary-plus.tsv")
-wgswxspanel_primary_file <- file.path(out_dir, 
-                              "independent-specimens.wgswxspanel.primary.tsv")
-wgswxspanel_relapse_file <- file.path(out_dir, 
-                              "independent-specimens.wgswxspanel.relapse.tsv")
-wgswxspanel_primplus_file <- file.path(out_dir, 
-                              "independent-specimens.wgswxspanel.primary-plus.tsv")
+wgs_primary_each_file <- file.path(out_dir, 
+                              "independent-specimens.wgs.primary.eachcohort.tsv")
+wgs_relapse_each_file <- file.path(out_dir, 
+                              "independent-specimens.wgs.relapse.eachcohort.tsv")
+wgs_primplus_each_file <- file.path(out_dir, 
+                              "independent-specimens.wgs.primary-plus.eachcohort.tsv")
+wgswxspanel_primary_each_file <- file.path(out_dir, 
+                              "independent-specimens.wgswxspanel.primary.eachcohort.tsv")
+wgswxspanel_relapse_each_file <- file.path(out_dir, 
+                              "independent-specimens.wgswxspanel.relapse.eachcohort.tsv")
+wgswxspanel_primplus_each_file <- file.path(out_dir, 
+                              "independent-specimens.wgswxspanel.primary-plus.eachcohort.tsv")
+
+wgs_primary_all_file <- file.path(out_dir, 
+                                   "independent-specimens.wgs.primary.allcohorts.tsv")
+wgs_relapse_all_file <- file.path(out_dir, 
+                                   "independent-specimens.wgs.relapse.allcohorts.tsv")
+wgs_primplus_all_file <- file.path(out_dir, 
+                                    "independent-specimens.wgs.primary-plus.allcohorts.tsv")
+wgswxspanel_primary_all_file <- file.path(out_dir, 
+                                           "independent-specimens.wgswxspanel.primary.allcohorts.tsv")
+wgswxspanel_relapse_all_file <- file.path(out_dir, 
+                                           "independent-specimens.wgswxspanel.relapse.allcohorts.tsv")
+wgswxspanel_primplus_all_file <- file.path(out_dir, 
+                                            "independent-specimens.wgswxspanel.primary-plus.allcohorts.tsv")
```

Comment:
For backward compatibility, could you change the all-cohorts filenames to be the same as previous results?

```suggestion
wgs_primary_all_file <- file.path(out_dir, 
                                   "independent-specimens.wgs.primary.tsv")
wgs_relapse_all_file <- file.path(out_dir, 
                                   "independent-specimens.wgs.relapse.tsv")
wgs_primplus_all_file <- file.path(out_dir, 
                                    "independent-specimens.wgs.primary-plus.tsv")
wgswxspanel_primary_all_file <- file.path(out_dir, 
                                           "independent-specimens.wgswxspanel.primary.tsv")
wgswxspanel_relapse_all_file <- file.path(out_dir, 
                                           "independent-specimens.wgswxspanel.relapse.tsv")
wgswxspanel_primplus_all_file <- file.path(out_dir, 
                                            "independent-specimens.wgswxspanel.primary-plus.tsv")
```